src/compiler/binder.ts

```diff
@@ -3386,12 +3382,15 @@ namespace ts {
     }
 
     function computePropertyDeclaration(node: PropertyDeclaration, subtreeFlags: TransformFlags) {
-        // A PropertyDeclaration is TypeScript syntax.
-        let transformFlags = subtreeFlags | TransformFlags.AssertTypeScript;
+        let transformFlags = subtreeFlags;

```

Comment:
I'm concerned about not having a `TransformFlag` for class fields, see my comment in the *classProperties.ts* file for reference.